server/plugin.go

```diff
@@ -116,15 +120,11 @@ func (p *Plugin) OnActivate() error {
 		p.configuration.SetDefaults()
 	}
 
-	if err = p.configuration.IsValid(); err != nil {
-		return fmt.Errorf("could not validate config: %w", err)
-	}
-
 	// initiate local tsdb
 	p.tsdbLock.Lock()
 	defer p.tsdbLock.Unlock()
 	p.db, err = tsdb.Open(*p.configuration.DBPath, p.logger, nil, &tsdb.Options{
-		RetentionDuration:              int64(30 * 24 * time.Hour / time.Millisecond),
+		RetentionDuration:              int64(localRetentionDays * 24 * time.Hour / time.Millisecond),
```

Comment:
@agnivade reduced local retention (the plugin storage) to `3` days. It's not necessary to have extended period of retention as we handle that in the MM filestore.